pennylane/ops/op_math/controlled_ops.py

```diff
@@ -117,68 +123,75 @@ class ControlledQubitUnitary(ControlledOp):
 
     @classmethod
     def _unflatten(cls, data, metadata):
-        return cls(
-            data[0], control_wires=metadata[0], control_values=metadata[1], work_wires=metadata[2]
-        )
+        return cls(data[0], metadata[0], control_values=metadata[1], work_wires=metadata[2])
 
     # pylint: disable=arguments-differ, too-many-arguments, unused-argument, too-many-positional-arguments
     @classmethod
     def _primitive_bind_call(
         cls,
         base,
-        control_wires: WiresLike,
-        wires: WiresLike = (),
+        wires: WiresLike,
+        control_wires: WiresLike = "unset",
         control_values=None,
         unitary_check=False,
         work_wires: WiresLike = (),
     ):
-        wires = Wires(() if wires is None else wires)
+        cls._deprecate_control_wires(control_wires)
+
         work_wires = Wires(() if work_wires is None else work_wires)
 
         if hasattr(base, "wires") and len(wires) != 0:
             warnings.warn(
                 "base operator already has wires; values specified through wires kwarg will be ignored."
             )
-            wires = Wires(())
+            # wires = Wires(())
 
-        all_wires = control_wires + wires
         return cls._primitive.bind(
-            base, control_wires=all_wires, control_values=control_values, work_wires=work_wires
+            base, wires=wires, control_values=control_values, work_wires=work_wires
         )
 
+    @classmethod
+    def _deprecate_control_wires(cls, control_wires):
+        if control_wires != "unset":
+            warnings.warn(
+                "The control_wires input to ControlledQubitUnitary is deprecated and will be removed in v0.42. "
+                "Instead, please use wires, following wires=controlled_wires+target_wire.",
```

Comment:
```suggestion
                "The control_wires input to ControlledQubitUnitary is deprecated and will be removed in v0.42. "
                "Please note that the second positional arg of your input is going to be the new wires, following wires=controlled_wires+target_wires, where target_wires is the optional arg wires in the legacy interface.",
```
Since this modification involves a drastic change on public UI side I believe it will be better to make the warning info much sharper.